src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -321,9 +321,19 @@ public AdsBaseObject verifyLineItemId(String slotName, int lineItemId) {
 
   public AdsBaseObject verifyAdImage(String slotName, String src, String imageUrl) {
     WebElement element = getIframe(slotName, src);
-    Assertion
-        .assertTrue(new AdsComparison().compareImageWithScreenshot(imageUrl, element, driver));
-    PageObjectLogging.log("verifyAdImage", "Ad looks good", true, driver);
+
+    try {
+      Assertion
+          .assertTrue(new AdsComparison().compareImageWithScreenshot(imageUrl, element, driver));
+      PageObjectLogging.log("verifyAdImage", "Ad looks good", true, driver);
+    } catch (AssertionError e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)